### ISSUE TITLE
how to hide YOUR_CLIENT_SECRET from my app

### ISSUE THREAD

**Issue by rupamking1** [hitherejoe/FlutterOAuth#19]
how to hide YOUR_CLIENT_SECRET from my app

**Comment by Ahmadre** [comment]
use: 

`--dart-define=SECRET=Value`

as your env-variable.

**Comment by rupamking1** [comment]
> use:
> 
> `--dart-define=SECRET=Value`
> 
> as your env-variable.

can i hide completely secret key from my app using this method?

**Comment by Ahmadre** [comment]
> > use:
> 
> > 
> 
> > `--dart-define=SECRET=Value`
> 
> > 
> 
> > as your env-variable.
> 
> 
> 
> can i hide completely secret key from my app using this method?

Yes, of course. Seems like you never heard of env-variables. They appear nowhere, check this out: [stackoverflow](https://stackoverflow.com/a/61725261) and please close this, because this hasn't anything to do with the package :)

**Comment by rupamking1** [comment]
is that works on release mode.

**Comment by Ahmadre** [comment]
> is that works on release mode.

Of course! It's especially made for releases and flavoured apps. Why don't you just try ;)
